### PR TITLE
added overview

### PR DESCRIPTION
**Pull request by tushgaurav**
added an overview with the list of technologies used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,3 +245,22 @@ Branch names are enforced before you even commit to ensure everyone adopts the s
 For end to end testing, we use the notorious [cypress](https://github.com/cypress-io/cypress).
 
 We also have a pre-built CI toolkit for you that will build and run the tests.
+
+# Overview
```
Comment (yann510) [inline README.md:249]: The overview should at the top just after the prerequisites.

File: README.md [hunk 2]
```diff
@@ -245,3 +245,22 @@ Branch names are enforced before you even commit to ensure everyone adopts the s
...
+
+Stator is a full-stack pre-configured template for your projects.
+The technologies used are mentioned below-
```
Comment (yann510) [inline README.md:252]:
```suggestion
The technologies used are mentioned below:
```

File: README.md [hunk 3]
```diff
@@ -245,3 +245,22 @@ Branch names are enforced before you even commit to ensure everyone adopts the s
...
+* Backend
+    * TypeORM
+    * Nest
+    * Fastify
+    * NestJs CRUD
+* Frontend
+    * React
+    * Redux Toolkit
+    * React Router
+    * axios
```
Comment (yann510) [inline README.md:266]: I think it would be great to add links to all the projects.

**Review by yann510 — CHANGES_REQUESTED** [review]
Thanks for the help bud, few improvements can be made.
Also, I'd be interested as to how you found the project and how you plan to use it :)

File: README.md (hunk 3 above)
Comment (tushgaurav) [reply README.md:266]: Okay, working on the suggestions.

**Review by tushgaurav — COMMENTED** [review]
[x] Changed the position of overview form bottom to after prerequisites.
[x] Added links to the technologies used list in the overview section.

**Review by tushgaurav — COMMENTED** [review]
[x] Added links to the list of technologies used.
[x] Changed the position of the overview section to after prerequisites.

File: README.md [hunk 4]
```diff
@@ -34,6 +34,24 @@ Of course, you won't be creating a todo application for your project, but you ca
 - [Docker Compose](https://docs.docker.com/compose/install/)
 - [node.js](https://nodejs.org/en/download/) your version must >= 14
 
+# Overview
```
Comment (yann510) [inline README.md:37]:
```suggestion
## Overview
```

To be consistent

File: README.md [hunk 5]
```diff
@@ -34,6 +34,24 @@ Of course, you won't be creating a todo application for your project, but you ca
...
+
+Stator is a full-stack pre-configured template for your projects.
+The technologies used are mentioned below:
+* Database
```
Comment (yann510) [inline README.md:41]: Use `###` instead of `*` for this level, this way you won't have a nested list.

**Review by yann510 — CHANGES_REQUESTED** [review]
Just thought about those, also leaving a star would help the project and I'd appreciate it if you could answer my previous questions: How you found the project and how you plan to use it?

**Comment by yann510** [comment]
Also, we should add a global section to list the global technologies.

**Comment by tushgaurav** [comment]
> Just thought about those, also leaving a star would help the project and I'd appreciate it if you could answer my previous questions: How you found the project and how you plan to use it?

I was searching for some projects to contribute to and that's how I found Stator.

**Review by yann510 — APPROVED** [review]
Awesome, thanks for your help.

**Pull request merged**